### PR TITLE
docs: Update manual testing doc: consumer first; separate halves; rm settings

### PR DESCRIPTION
**Pull request by timmc-edx**
- Split out producer/consumer testing into separate sections
- Put consumer before producer to increase chance of success.
  (Mention topic creation issue.)
- Remove Django settings -- these are now in `cms/envs/devstack.py`
- Put both rsync mentions together
- Fix top heading

**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Version bumped
- [ ] Changelog record added
- [x] Documentation updated (not only docstrings)
- [x] Commits are squashed
- [x] Noted any: Concerns, dependencies, deadlines, tickets, testing instructions

### CODE REVIEW

File: docs/how_tos/manual_testing.rst [hunk 1]
```diff
@@ -6,30 +6,24 @@ The producer can be tested manually against a Kafka running in devstack.
...
 #. Load the control center UI: http://localhost:9021/clusters and wait for the cluster to become healthy
-#. In edx-platform's ``cms/envs/common.py``:
-
-   - Add ``'edx_event_bus_kafka'`` to the ``INSTALLED_APPS`` list
-   - Add the following::
-
-       EVENT_BUS_KAFKA_CONSUMERS_ENABLED = True
-       EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS = "edx.devstack.kafka:29092"
-       EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL = "http://edx.devstack.schema-registry:8081"
+#. In edx-platform's ``cms/envs/common.py``, add ``'edx_event_bus_kafka'`` to the ``INSTALLED_APPS`` list
```
Comment (rgraber) [inline docs/how_tos/manual_testing.rst:9]: I don't think this is necessary. I never do it.

File: docs/how_tos/manual_testing.rst [hunk 2]
```diff
@@ -6,30 +6,24 @@ The producer can be tested manually against a Kafka running in devstack.
...
    - Go to the topic that was created and then into the Messages tab; select offset=0 to make sure you can see messages that were sent before you had the UI open.
 
-#. Test the consumer:
-
-   - Run the example command listed in the ``edx_event_bus_kafka.consumer.event_consumer.ConsumeEventsCommand`` docstring
-   - Expect to see output that ends with a line containing "Received SESSION_LOGIN_COMPLETED signal with user_data"
-   - Kill the management command (which would run indefinitely).
-
 #. Rerun rsync after any edits as needed.
+#. When you're done testing, find the consumer process with ``jobs`` and terminate it with ``kill``, since it would otherwise run indefinitely.
```
Comment (rgraber) [inline docs/how_tos/manual_testing.rst:27]: Once we fix the Exception/BaseException thing regular ctrl-c should work

**Review by rgraber — CHANGES_REQUESTED** [review]
I think it would be clearer if we move 'test the consumer' to a separate section and just update it to be
1. run command
2.  do everything we told you in the producer
3. check output

File: docs/how_tos/manual_testing.rst (hunk 1 above)
Comment (timmc-edx) [reply docs/how_tos/manual_testing.rst:9]: Oh, no -- it's necessary for running the management commands, which is the approach these instructions use.

File: docs/how_tos/manual_testing.rst (hunk 2 above)
Comment (timmc-edx) [reply docs/how_tos/manual_testing.rst:27]: Verified!

**Review by rgraber — APPROVED** [review]
LGTM

**Pull request merged**